src/NewPost.js

```diff
@@ -0,0 +1,70 @@
+import React, { Component } from "react";
+
+export class NewPost extends Component {
+  state = {
+    title: "",
+    author: "",
+    createdOn: "",
+    body: "",
+    tags: []
+  };
+
+  handleChange = e => {
+    this.setState({
+      [e.target.name]: e.target.value
+    });
+  };
+
+  handleSubmit = e => {
```

Comment:
try logging the `state` here so you can verify that the different values are getting set properly